app/src/main/java/fr/neamar/kiss/broadcast/InstallShortcutHandler.java

```diff
@@ -61,7 +61,7 @@ public void onReceive(Context context, Intent data) {
             }
         }
                 
-        dh.getShortcutProvider().addShortcut(pojo);
+        dh.addShortcut(pojo);
```

Comment:
The launcher needs to keep track of the installed shortcuts even if it does not display them. The shortcut list is not actually stored in memory (well it is copied into the I/O cache, but that doesn't really count…): There is just some data added to or deleted from it whenever a corresponding broadcast is received.